Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -267,11 +290,22 @@ bool CBoot::SetupWiiMemory(IOS::HLE::IOSC::ConsoleType console_type)
     {
       gen.SetBytes(std::move(data));
       serno = gen.GetValue("SERNO");
+      model = gen.GetValue("MODEL");
       if (SConfig::GetInstance().bOverrideRegionSettings)
       {
         region_setting = RegionSetting{gen.GetValue("AREA"), gen.GetValue("VIDEO"),
                                        gen.GetValue("GAME"), gen.GetValue("CODE")};
       }
+      else
+      {
+        const std::string code = gen.GetValue("CODE");
+        if (code.size() >= 2 && CodeRegion(code[1]) == SConfig::GetInstance().m_region)
+          region_setting.code = code;
```

Comment:
`code` can be made non-const and moved here:

```suggestion
          region_setting.code = std::move(code);
```